### ISSUE TITLE
01. Fetch Villagers

### ISSUE THREAD

**Issue by cat-miller** [cat-miller/capstone-project#1]
### Value Description / Statement

As a *user*
I want to see *all villagers of Animal Crossing displayed as a list of cards*
so that I have *all the information I need* when I have to decide which villager should be my next neighbor. 

### Description
<img width="331" alt="UserStory01" src="https://user-images.githubusercontent.com/99614110/169715181-91fb1ae9-01da-42cb-bf3f-2e3a934ec87b.png">

### Acceptance Criteria

- [x]  Picture of villager should be displayed on card
- [x] Name of villager should be displayed on card
- [x] Species and gender should be displayed on card
- [x] There should be a list of cards

### Tasks

- [x] Create a new branch
- [x] Create a ```VillagerList``` on a ```VillagerPage```
- [x] fetch villagers data from ACNH API
- [x] Create component ```VillagerCard```
- [x] Make sure there a all villagers displayed

### Complexity

low

**Comment by CJAkkuec** [comment]
Hey there! Looking good, although I do have a couple notes:

I'm not sure if this should/would be your first User Story in terms of relevancy. As far as I remember, the core feature of your app is a journal. Your first step would be to make sure that the user can create and then as a next step display said journal entries. I'm assuming that eventually you'll want to be able to include the villagers into said journal. If the journal feature doesn't exist beforehand, there is no immediate functionality.

This follows the principle of "I want to have a working app every step of the way". The first step would be providing the user with a form to create journal entries, and then a page that displays those entries. That way, we've established the core functionality of the app. Once you've got the basics established, you can start including more features.

Perhaps (and this is just an example) you'll have another (initial) form that lets you create an overview over your island, which might be displayed over your journal entries. The data displayed could be the initial fruits and villagers on your island, etc. At that point you can start fetching the relevant data your user needs.

I completely understand that working with the ACNH API might be incredibly tempting off the bat (and I'd be just as tempted), but we need to make sure our core features are in working order before we can start throwing glitter at it.

Minor additional note: Perhaps consider being very specific with your naming. Something like "VillagerList", "VillagerCard", etc. That way other people can identify what's going on a little more quickly.

**Comment by cat-miller** [comment]
Dear CJ, 
thank you for your review. I know what you mean. But after thinking about it for a while, I decided to show the inhabitants in a list first. In the next steps, the user should be able to bookmark them and also get lists of the animals he can catch. In the end I came to the conclusion that it is more useful for the user if he can look up inhabitants and animals and remember them, than to create a "passport", which is nice to have and will be included by me later on. 

Thank you for reminding me to be more specific about the naming. This is indeed really important an I was quite overwhelmed with structuring and thinking about my capstone project..
